modules/tinymce/src/plugins/table/main/ts/queries/TableTargets.ts

```diff
@@ -18,11 +17,11 @@ const noMenu = (cell: SugarElement<HTMLTableCellElement | HTMLTableCaptionElemen
   selection: [ cell ]
 });
 
-const forMenu = (selections: Selections, table: SugarElement<HTMLTableElement>, cell: SugarElement<HTMLTableCellElement>): RunOperation.CombinedTargets => ({
+const forMenu = (selections: Selections, table: SugarElement<HTMLTableElement>, cell: SugarElement<HTMLTableCellElement>, ephemera: Ephemera): RunOperation.CombinedTargets => ({
```

Comment:
Does it make sense to pass `ephemera` here? We could just import it and use it directly, as we don't try to use multiple variants.

Reply:
Just realised I'm still exporting it as `Ephemera` too